.github/workflows/pull_request.yaml

```diff
@@ -38,36 +38,14 @@ jobs:
 
     - name: Lint
       run: |
-        go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.56.0
+        go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.61.0
         golangci-lint run --timeout=10m ./...
 
     - name: Helm Lint
       run: |
         cd deployments/kubernetes/chart/forecastle
         helm lint
 
-    # - name: Install kubectl
```

Comment:
Why commenting out this? Why not just remove it since it was move to run-tests-on-pr

Reply:
you are looking at a removed change :smile: